pandas/tests/io/test_sql.py

```diff
@@ -3333,6 +3333,25 @@ def test_roundtripping_datetimes(sqlite_sqlalchemy_memory_engine):
     assert result == "2020-12-31 12:00:00.000000"
 
 
+@pytest.fixture
+def sqlite_builtin_detect_types():
+    with contextlib.closing(
+        sqlite3.connect(":memory:", detect_types=sqlite3.PARSE_DECLTYPES)
+    ) as closing_conn:
+        create_and_load_iris_view(closing_conn)
```

Comment:
```suggestion
```

Doesn't look like you're actually using the iris data in the test?

Reply:
Yes, thanks, left-over from a copy paste